aicsimageio/exceptions.py

```diff
@@ -6,12 +8,12 @@ class UnsupportedFileFormatError(Exception):
     the supported file types and cannot be parsed with AICSImage.
     """
 
-    def __init__(self, type_, **kwargs):
+    def __init__(self, data, **kwargs):
         super().__init__(**kwargs)
-        self.type_ = type_
+        self.data = data
```

Comment:
stylistic `self._data` instead of `self.data`

Reply:
This is an exception type though, no user should ever be interacting with it no?